### PR TITLE
Add studyL Stephens_2023

### PR DESCRIPTION
**Pull request by ehwenk**
Add pollination syndrome & pollination system data from Ruby Stephens.

Report has been reviewed.

Ruby has also reviewed the trait definitions & trait values for pollination syndrome & pollination system.

### CODE REVIEW

File: config/traits.yml [hunk 1]
```diff
@@ -2586,34 +2586,53 @@ traits:
...
-        butterfly: Butterfly-pollinated
-        fly: Fly-pollinated
-        insect: Insect-pollinated, type of insect not specified
-        moth: Moth-pollinated
-        none: Spore-producing plants
-        vertebrate: Vertebrate-pollinated, broadly defined
-        wind: wind-pollinated
-        pollination_rare: plants mostly reproduce through vegetative spread and flowering is rare
+        abiotic: Floral syndrome indicates pollination occurs through wind, water, or collision of free-floating particles.
+        biotic: Floral syndrome suggests flowers are pollinated by a biotic agent.
```
Comment (dfalster) [inline config/traits.yml:2594]: These wordings are nice, but they suggest pollination will only ever be inferred from floral syndrome. What if someone observes pollination directly?

Comment (dfalster) [reply config/traits.yml:2594]: are, I see trait below has space for field measurements. Would this be a different trait, or rather different value type? When I see the names pollination_syndrome and pollination_system, it doesn't immediately suggest one is observed, the other is inferred.

File: data/Stephens_2023/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  custom_R_code: .na
+  collection_date: 2022/2023
+  taxon_name: canonicalName
+  trait_name: trait_name
+  value: value
+  description: Study that models the evolution of pollination across angiosperms and
+    quantify the timing and environmental associations of pollination shifts.
+  basis_of_record: literature
+  life_stage: adult
+  sampling_strategy: Scoring pollination mode ^p Pollination system or syndrome was
```
Comment (dfalster) [inline data/Stephens_2023/metadata.yml:40]: Text for sampling strategy Needs to go in quotes, as text contains a `:`

File: data/Stephens_2023/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
...len and floral rewards (Table 1). To separate insect from vertebrate pollination syndromes we considered floral size and the robustness of floral parts, nectar quantity and the accessibility of floral rewards to different pollinators (e.g. the presence of poricidal anthers which only release pollen when vibrated by bees in buzz-pollinated flowers (Pritchard & Vallejo-Marín, 2020), Table 1). Water pollination was considered in the rare cases where plants had an aquatic habit and flowered near or under water (Ackerman, 2000). ^p Flowers were scored as polymorphic where there was evidence for more than one pollination mode, or where they were pollinated by animals but it was unclear whether this was vertebrate or insect pollination (n = 76). Where there was no evidence that pollination via external vectors occurred (in clonal or autogamous species) or no information was available these species were left as missing data (n = 41). Our final dataset included pollination information for 1160 of the 1201 species in 433 families (all except Hoplestigmataceae) in the Ramírez-Barahona et al. (2020) angiosperm tree. Fully referenced data are available at https://doi.org/10.5281/zenodo.7592528.
```
Comment (dfalster) [inline data/Stephens_2023/metadata.yml:94]: Text for methods needs to go in quotes, as text contains a `:`

File: config/traits.yml (hunk 1 above)
Comment (ehwenk) [reply config/traits.yml:2594]: These are the definitions used within the field of pollination biology. See jeffollerton.co.uk/2019/09/05/what-exactly-is-a-pollination-system/

Also, Ruby confirmed these differences between traits are standard in the field and used the same distinction in her dataset.

File: data/Stephens_2023/metadata.yml (hunk 2 above)
Comment (ehwenk) [reply data/Stephens_2023/metadata.yml:40]: fixed...

File: data/Stephens_2023/metadata.yml (hunk 3 above)
Comment (ehwenk) [reply data/Stephens_2023/metadata.yml:94]: fixed

**Review by dfalster — APPROVED** [review]

**Pull request merged**